src/librustc_metadata/creader.rs

```diff
@@ -636,19 +651,22 @@ impl<'a> CrateReader<'a> {
             self.sess.span_fatal(item.span, &message);
         }
 
-        return ret
+        return ret;
     }
 
     /// Look for a plugin registrar. Returns library path, crate
     /// SVH and DefIndex of the registrar function.
-    pub fn find_plugin_registrar(&mut self, span: Span, name: &str)
+    pub fn find_plugin_registrar(&mut self,
+                                 span: Span,
+                                 name: &str)
```

Comment:
Is there a reason `rustfmt` gives each argument its own line here?
I think it just wastes vertical space without helping readability.